danger.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "git", "~> 1"
   spec.add_runtime_dependency "colored", "~> 1.2"
   spec.add_runtime_dependency "faraday", "~> 0"
+  spec.add_runtime_dependency "faraday-http-cache", "~> 1.0"
```

Comment:
I went this way to also get caching of GitHub API calls that are made from within the Dangerfile or plugins. Figured there is enough logic in how to properly cache that I didn't want to duplicate it.